vm-virtio/src/lib.rs

```diff
@@ -124,6 +124,7 @@ impl fmt::Display for VirtioDeviceType {
 const INTERRUPT_STATUS_USED_RING: u32 = 0x1;
```

Comment:
nit: typo in your commit message s/mut not deliver/must not deliver/

Reply:
Thanks! Update is done.